pkg/sfu/audio/audiolevel.go

```diff
@@ -65,7 +67,12 @@ func NewAudioLevel(params AudioLevelParams) *AudioLevel {
 }
 
 // Observes a new frame, must be called from the same thread
-func (l *AudioLevel) Observe(level uint8, durationMs uint32) {
+func (l *AudioLevel) Observe(level uint8, durationMs uint32, arrivalTime time.Time) {
+	l.lock.Lock()
+	defer l.lock.Unlock()
+
```

Comment:
While doing this, had a call to `resetIfStaleLocked` here, but later removed it as I wanted to avoid it for every packet. `GetActive` would reset if the samples are stale. if `GetActive` is not called, this could be looking at a longer window, but the activity thresholds should kick in.

If there is a need to check, can add the stale check here, but good to avoid per packet.

Reply:
I forgot to remove the `arrivalTime` argument after I removed the call to `resetifStaleLocked`. But, I think I am going to leave that argument in there as unused.